### ISSUE TITLE
Bumblebee Install Pegging Cores

### ISSUE THREAD

**Issue by shantanu-vyas** [Bumblebee-Project/Bumblebee#844]
I installed the non-free nvidia bumblebee drivers today and upon restarting noticed my computer would instantly peg all 8 cores and the core temp got extremely hot (95c idle at 40). I looked into it and it was bumblebee using gcc to build something. I think the installer might request all avail cores and caused this to happen.

**Comment by bluca** [comment]
Depending on the platform and the drivers packaging, the out-of-tree proprietary nvidia kernel modules might only get compiled by DKMS for your running kernel the first time they are loaded at boot.
I can't think of anything else that would cause bumblebee to run gcc at boot.

In any case it's not really anything to do with bumblebee, if you were running on a desktop computer the same thing would have happened.

**Comment by gsgatlin** [comment]
Is this on the fedora or CentOS / RHEL distributions? If so it is caused by bumblebee-nvidia service and only happens when a new kernel update happens or after a fresh install of bumblebee-nvidia rpm. This is caused by nvidia installer program.

**Comment by shantanu-vyas** [comment]
Yeah that was it sorry I forgot to mention, it was after a kernel update Fedora 25. It just took me by surprise that my cores went up to 95c~ during the installer.
